sopel/tools/target.py

```diff
@@ -52,12 +55,16 @@ def __init__(self, name):
         self.users = {}
         """The users in the channel.
 
-        This maps username ``Identifier``\\s to channel objects."""
+        This maps username :class:`Identifier <sopel.tools.Identifier>`\\s to
+        :class:`User` objects.
+        """
         self.privileges = {}
         """The permissions of the users in the channel.
 
-        This maps username ``Identifier``s to bitwise integer values. This can
-        be compared to appropriate constants from ``sopel.module``."""
+        This maps username :class:`Identifier <sopel.tools.Identifier>`\\s to
+        bitwise integer values. This can be compared to appropriate constants
+        from :class:`sopel.module`.
```

Comment:
This isn't a class, but a module:

```suggestion
        from :mod:`sopel.module`.
```

Reply:
Sphinx doesn't actually care (the cross-ref works even though it's not a class)—but you're right, we should use the correct reference domain anyway.